storage/engine/rocksdb/db.cc

```diff
@@ -1353,15 +1353,9 @@ DBStatus DBOpen(DBEngine **db, DBSlice dir, DBOptions db_opts) {
   // increased read amplification.
   table_options.block_size = db_opts.block_size;
 
-  rocksdb::ColumnFamilyOptions cf_options;
-  cf_options.OptimizeLevelStyleCompaction(db_opts.memtable_budget);
-  // OptimizeLevelStyleCompaction sets no-compression for L0 and
-  // L1. Current benchmarks and tests show no benefit to doing this.
-  for (int i = 0; i < cf_options.compression_per_level.size(); i++) {
-    cf_options.compression_per_level[i] = rocksdb::kSnappyCompression;
-  }
-
-  rocksdb::Options options(rocksdb::DBOptions(), cf_options);
+  rocksdb::Options options(rocksdb::GetOptions(db_opts.memtable_budget));
+  options.IncreaseParallelism(db_opts.num_cpu);
```

Comment:
Which line? The `IncreaseParallelism` call allows multiple background compactions to be performed concurrently and ties into the `max_subcompactions` setting below which will use multiple threads to speed up a single compaction.